Source/Factors/OktaFactor.swift

```diff
@@ -150,6 +150,7 @@ open class OktaFactor {
 
     public func verify(passCode: String?,
                        answerToSecurityQuestion: String?,
+                       rememberDevice: Bool? = nil,
```

Comment:
Thank you for assigning a default; it's nice that this won't result in a breaking API change.